src/urllib3/__init__.py

```diff
@@ -80,3 +82,28 @@ def disable_warnings(category=exceptions.HTTPWarning):
     Helper for quickly disabling all urllib3 warnings.
     """
     warnings.simplefilter("ignore", category)
+
+
+# PoolManager for a top-level request method.
+# DEFAULT_POOL is a module global PoolManager instance,
+# therefore its side effects could be shared across dependencies relying on it.
+# To avoid side effects create a new PoolManager instance and use it instead.
+try:
+    import certifi
```

Comment:
I'm not thrilled about using `certifi` as a default given us trying to move away from it and to instead rely on system certificates. We use system certificates by default on Linux and kinda on Windows and not on macOS iirc. Not certain what to do about this since whatever we choose we're supporting for all of 2.x.

Reply:
Initially I used certifi because of the #1993 description, but I think it would be better to use system certificates by default. Removed certifi.